composer.json

```diff
@@ -15,17 +15,17 @@
     }
   },
   "require": {
-    "php": ">=5.6"
+    "php": "^7.1"
   },
   "require-dev": {
-    "squizlabs/php_codesniffer": "^3.3",
+    "squizlabs/php_codesniffer": "^3.5",
     "roave/security-advisories": "dev-master",
-    "phpunit/phpunit": "^5.7",
-    "php-coveralls/php-coveralls": "^2.1"
+    "phpunit/phpunit": "^7.5",
+    "php-coveralls/php-coveralls": "^2.2"
   },
   "config": {
     "platform": {
-      "php": "5.6"
+      "php": "7.1"
```

Comment:
this whole platform config can be deleted I suppose. I used it before so php 5 compatible deps would be installed

Reply:
done